packages/libs/components/src/map/DonutMarker.tsx

```diff
@@ -266,31 +273,32 @@ function donutMarkerSVGIcon(props: DonutMarkerStandaloneProps): {
         // only sum up in cumulative mode
         cumulativeSum += thisValue;
 
-      //DKDK draw arc: makeArc(centerX, centerY, Radius for arc, start point of arc (radian), end point of arc (radian))
+      // draw arc: makeArc(centerX, centerY, Radius for arc, start point of arc (radian), end point of arc (radian))
       svgHTML +=
         '<path fill="none" stroke="' +
         (el.color ?? 'silver') +
         '" stroke-width="4" d="' +
```

Comment:
What do you think about setting `stroke-width` to be 5? It was kinda nice to see more of the data colors, but perhaps there was a reason to set it to 4 originally?

(with stroke-width=5)
<img width="566" alt="Screen Shot 2023-04-27 at 5 49 34 AM" src="https://user-images.githubusercontent.com/11710234/234826603-fed6c06f-95b3-48d5-ad3b-da291bb5c3f5.png">

Reply:
@asizemore I don't think we have a specific reason not to use larger stroke-width as you suggested. 🤔 Will ask Bob about this at slack, just in case.

Reply:
I like the bolder look. The only issue is squeezing in the numbers. I think above 9999 we switch to 10k, 11k etc, so we should be fine up to 999k. Looks like room for 4 characters?